src/cpp/cpp_typecheck_fargs.cpp

```diff
@@ -109,7 +109,8 @@ bool cpp_typecheck_fargst::match(
       return false;
   }
 
-  for(std::size_t i=0; i<ops.size(); i++)
+  exprt::operandst::iterator it=ops.begin();
+  for(std::size_t i=0; i<ops.size(); i++, ++it)
```

Comment:
Still mixing post fix and prefix incrementing. Maybe this isn't really a problem (I think it looks weird but it isn't in the coding guidelines) but I thought I'd see this being picked up in other PR reviews.